README.md

```diff
@@ -1,15 +1,41 @@
+![image](https://github.com/scale-it/algo-builder/blob/master/media/logo.svg) 
 # Algo Builder Templates
 
 Dapps templates for [Algo Builder](https://github.com/scale-it/algo-builder). Use these templates to quickly setup/get started with your Dapp implementation.
 
+Distributed Applications or dApps are basically decentralized applications. The templates in this repository are extremely helpful and easy to use. They are designed and implemented in such a way that they can provide the aspiring developers a headstart in building dApps based on `Algorand Blockchain`. The templates heavily use [algosdk](https://github.com/algorand/js-algorand-sdk), [algo-builder](https://github.com/scale-it/algo-builder) and [Algosigner](https://github.com/PureStake/algosigner).
+
+The detailed description about each template and how t prperly use them can be found in their respective folders.
+
 ## Usage
 Each template is a single project (with 1 package.json file) comprising of the following:
- - An `algob` project : Used to deploy ASA's, stateless and stateful contracts using algob `deployer` object. Use `algob deploy` to deploy your `/scripts`. Sample algob project can  be found [here](https://github.com/scale-it/algo-builder/tree/master/packages/algob/sample-project), check the project readme.md for more details.
+ - An `algob` project : Used to deploy ASA's, stateless and stateful contracts using algob `deployer` object. Use `algob deploy` to deploy your `/scripts`. Sample algob project can  be found [here](https://github.com/scale-it/algo-builder/tree/master/packages/algob/sample-project), check the project `README.md` for more details.
```

Comment:
please use a link for `README.md` instead of code marks.

Reply:
okay